src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -48,7 +50,11 @@
     private static final String sqlQuery = "SELECT GENERATE_UUID() uuid, CURRENT_TIMESTAMP() as example_now" +
         " FROM UNNEST(GENERATE_ARRAY(1, 3));";
 
-    private final Map<String, String> columnReplacements = Map.of("example_now", "example:identifier.now");
+    private static final String searchQuery = "{\"query_string\": {\"query\": \"([example:identifier.now]:0)\"}}";
+
+    private static final Map<String, String> columnReplacements = new ImmutableMap.Builder<String, String>()
```

Comment:
Any reason to use an immutable map here and not just `Map.of`?

Reply:
I replaced an immutable BiMap earlier, so I used the same builder construction.